recipe/meta.yaml

```
@@ -33,23 +33,25 @@ requirements:
   host:
     - python
     - pip
-    - setuptools
+    - setuptools >=61.0.0
     - wheel
     - numpy {{ numpy }}
     - packaging
     - py-cpuinfo
-    - cython 0.29
+    - cython >=0.29.32
     - blosc 1.21.3
     - c-blosc2 2.12.0
     - hdf5 1.12.1
     - lzo 2.10
     - bzip2 1.0.8
```

```suggestion
    - blosc 1.21.3
    - c-blosc2 2.12.0
    - hdf5 {{ hdf5 }}
    - lzo {{ lzo }}
    - bzip2 {{ bzip2 }}
```
Because packages are mentioned in a generic cbc.yaml, see https://github.com/AnacondaRecipes/aggregate/blob/master/conda_build_config.yaml#L12
It will help to maintain the recipe in the future